uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/toil-request-ctrl_test.js

```diff
@@ -277,6 +280,25 @@
               it('sets new expiry date', function () {
                 expect($ctrl.request.toil_expiry_date).toEqual(newExpiryDate);
               });
+
+              describe('and staff edits', function() {
+                beforeEach(function () {
+                  var directiveOptions = {
+                    contactId: toilRequest.contact_id, //owner's contact id
+                    leaveRequest: $ctrl.request
+                  };
+
+                  initTestController(directiveOptions);
+                });
+
+                it('has expired date set by manager', function() {
+                  expect($ctrl.request.toil_expiry_date).toEqual(newExpiryDate);
```

Comment:
newExpiryDate is set by manager in parent describe and checking if when creating controller for staff if it still has the date or changes it (bug fix)